runtimes/mainnet/src/lib.rs

```diff
@@ -1367,6 +1367,7 @@ impl thea_executor::Config for Runtime {
 	type Swap = AssetConversion;
 	type MultiAssetIdAdapter = AssetId;
 	type AssetBalanceAdapter = u128;
+	type GovernanceOrigin = EnsureRootOrHalfCouncil;
```

Comment:
Please extend pallets' mocks implementations with this type definition as well. CI is failing because of this...